lib/question.js

```diff
@@ -579,10 +579,11 @@ module.exports = {
 
                     /* If the submission was marked invalid during grading the grading job will
                        be marked ungradable and we should bail here to prevent LTI updates. */
-                    grading_job = result.rows[0];
-                    if (!grading_job.gradable) return callback(new NoSubmissionError());
-
-                    debug('_gradeVariantWithClient()', 'inserted', 'grading_job.id:', grading_job.id);
+                    grading_jobs = result.rows;
+                    for (const grading_job of grading_jobs) {
+                        if (!grading_job.gradable) return callback(new NoSubmissionError());
```

Comment:
Hmmmmmm... Just a question about this @bevansm . Do we know if all of the grading_jobs would be gradable or not at this point? 
 
Let's suppose external grading jobs are running at the same time. Do we have to worry about the race condition here?

Reply:
At this point, we can have only a manual grading job or an external grading job per variant. Hence, it falls under the same logic as before, as we can't actually have multiple types of jobs, and hence multiple jobs, per question in production.

+ gradable status currently only relates to external jobs--all manual jobs should be marked as gradable. If this needs to change for grading pipelines (i.e. only mark a manual job as gradable once external jobs have finished), then this might have a different meaning.